cmake/SeasocksConfig.cmake.in

```diff
@@ -0,0 +1,14 @@
+set(SEASOCKS_SHARED @SEASOCKS_SHARED@)
+set(SEASOCKS_STATIC @SEASOCKS_STATIC@)
+set(SEASOCKS_DEFLATE_SUPPORT @DEFLATE_SUPPORT@)
+
+if(SEASOCKS_STATIC)
+    find_package(Threads REQUIRED)
+    if(SEASOCKS_DEFLATE_SUPPORT)
+        find_package(ZLIB REQUIRED)
+    endif()
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/SeasocksTargets.cmake")
+
+@PACKAGE_INIT@
```

Comment:
This is a feature of the macros of https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html
I don't use it, but I added it just in case.
What it basically does is add a `set_and_check` macro, to avoid having paths of the build machines exposed in the installed package.

Reply:
Got it: thanks.